inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -22,6 +22,10 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
         InferenceEngine::Precision::FP16
 };
 
+const std::vector<InferenceEngine::Precision> intPrecisions = {
```

Comment:
Why cannot we combine int precision with input or net precisions?

Reply:
The idea is to have explicit test for two different implementations (rounding).

Reply:
Also not all of the activationTypes map support integer types.